internal/api/mfa.go

```diff
@@ -97,7 +97,7 @@ func (a *API) EnrollFactor(w http.ResponseWriter, r *http.Request) error {
 	}
 
 	if len(factors) >= int(config.MFA.MaxEnrolledFactors) {
-		return forbiddenError("Enrolled factors exceed allowed limit, unenroll to continue")
+		return unprocessableEntityError(ErrorCodeTooManyEnrolledMFAFactors, "Enrolled factors exceed allowed limit, unenroll to continue")
```

Comment:
Potential legitimate HTTP status code change, though unlikely to be handled by developers.

Reply:
Should be fine.